src/js/player.js

```diff
@@ -4290,12 +4291,24 @@ class Player extends Component {
    * @return {string}
    *         The current language code when getting
    */
-  language(code) {
+  language(code, skipListeners) {
     if (code === undefined) {
       return this.language_;
     }
 
-    this.language_ = String(code).toLowerCase();
+    if (this.language_ !== String(code).toLowerCase()) {
+      this.language_ = String(code).toLowerCase();
+      // skipListeners helps to avoid trigger on initialization  when component is not evented
+      if (!skipListeners) {
```

Comment:
I don't think I understand the need for this... if the component is not evented, it will not have a `languagechange` listener, so there should be no harm done by always triggering the event.

Reply:
We want this because in the first call when we initialized the language the component is not evented when this happens and will throw a `trigger` function is not defined.

Reply:
Gotcha, I get it now! 👍